site/docs/operations/docker.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Hazelcast Jet with Docker
+description: How to use Jet in a Docker environment
+---
+
+This section provides comprehensive coverage of usage of Hazelcast Jet
+inside a Docker environment. While it gives an explanation of all
+options and parameters used in the examples it is recommended to be
+familiar with Docker, visit e.g.  the [official
+documentation](https://docs.docker.com/get-started/) for a beginner
+focused tutorial.
+
+## Start Hazelcast Jet node
+
+Let's begin with starting an instance of Hazelcast Jet using the
+official Docker image. To start the instance simply execute the
+following command in your terminal:
+
+```bash
+docker run -p 5701:5701 hazelcast/hazelcast-jet
+```
+
+If you previously haven't run Hazelcast Jet Docker image, it will
+download the latest version of the image from [Docker
+hub](https://hub.docker.com/r/hazelcast/hazelcast-jet/). You can check
+the version of Jet in the logs, look for a line similar to this one:
+
+```text
+2020-05-18 19:34:43,872 [ INFO] [main] [c.h.system]: Hazelcast Jet {jet-version} (20200429 - e6c60a1) starting at [172.17.0.2]:5701
+```
+
+You can also verify the version of Hazelcast Jet by running a following
+command without starting a full instance:
+
+```bash
+docker run hazelcast/hazelcast-jet jet --version
+```
+
+You should get output similar to:
+
+```text
+$ docker run hazelcast/hazelcast-jet jet --version
+Hazelcast Jet {jet-version}
+Revision e6c60a1
+Build 20200429
+```
+
+If your `hazelcast/hazelcast-jet` latest image points to an older
+version you can update it by running the following command:
+
+```bash
+docker pull hazelcast/hazelcast-jet
+```
+
+You can also specify a particular version of the image to run when
+starting the container, e.g. when you need to run an older version of
+Jet:
+
+```bash
+docker run hazelcast/hazelcast-jet:4.0
+```
+
+## Submit job to Hazelcast Jet
+
+We will submit a hello world job, which is available in the Hazelcast
+Jet distribution. Download the Jet distribution from
+[here](https://github.com/hazelcast/hazelcast-jet/releases/download/v{jet-version}/hazelcast-jet-{jet-version}.tar.gz)
+. From now and on we will refer to the extracted location as `<jet home>`.
+
+Also, note the address of your Hazelcast Jet instance from the log:
+
+```text
+Members {size:1, ver:1} [
+    Member [172.17.0.2]:5701 - c54413e4-2cd2-4c07-91ec-d5992b18f9b0 this
+]
+```
+
+Run the following command to submit the hello world job to your Hazelcast
+Jet instance:
+
+```bash
+docker run -it -v "$(pwd)"/examples:/examples hazelcast/hazelcast-jet jet -a 172.17.0.2 submit /examples/hello-world.jar
+```
+
+The parameters are as follows:
+
```

Comment:
Let's make the parameter texts below monospaced to ease the reading experience, e.g., `-it`, `-v "$(pwd)"/examples:/examples`.